File: feature-indexer/models/service.py
class Service(object):
    def __init__(self, id, wfs_url, title, description, publisher, geometry, features):
        self.id = id
        self.wfs_url = wfs_url
        self.title = title
        self.description = description
        self.publisher = publisher
        self.geometry = geometry
        self.features = features

    def __str__(self):
        return self.__dict__

    def __repr__(self):
        return self

    def serialize(self):
        return {
            'id': self.id,
            'wfs_url': self.wfs_url,
            'title': self.title,
            'description': self.description,
            'publisher': self.publisher,
            'geometry': self.geometry,
            'features': self.features,
        }

File: service-finder/src/main/java/br/com/helisson/servicefinder/domain/Service.java
package br.com.helisson.servicefinder.domain;

import lombok.Data;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import java.math.BigDecimal;
import java.util.Date;
import java.util.List;

@Data
@Entity
public class Service {
    @Id
    private String id;

    @Column
    private String wfsUrl;

    @Column
    private String wmsUrl;

    @Column
    private String serviceProcessed;

    @Column
    private String title;

    @Column
    private String description;

    @Column
    private String publisher;

    @Column
    private String registerId;

    @Column
    private Date startDate;

    @Column
    private Date endDate;

    @Column
    private BigDecimal area;

    @Column
    private BigDecimal xMin;

    @Column
    private BigDecimal yMin;

    @Column
    private BigDecimal xMax;

    @Column
    private BigDecimal yMax;

    @OneToMany(mappedBy = "service")
    private List<FeatureType> features;
}
